ledger/artifactmanager/handler.go

```diff
@@ -55,6 +56,7 @@ func (h *MessageHandler) Init(ctx context.Context) error {
 	h.Bus.MustRegister(core.TypeGetObject, h.messagePersistingWrapper(h.handleGetObject))
 	h.Bus.MustRegister(core.TypeGetDelegate, h.messagePersistingWrapper(h.handleGetDelegate))
 	h.Bus.MustRegister(core.TypeGetChildren, h.messagePersistingWrapper(h.handleGetChildren))
+	h.Bus.MustRegister(core.TypeGetHistory, h.messagePersistingWrapper(h.handleGetHistory))
```

Comment:
It seems that It's not a good idea to add logic of blockexploler to the handlers. I suppose It should be discussed